archimedes/Makefile

```diff
@@ -0,0 +1,16 @@
+include ../make-base/stubs.mk
+
+app_name=archimedes
+include ../make-base/docker-compose.mk
+
+app_name_capitalized:=$(shell echo ${app_name} | tr [a-z] [A-Z])
```

Comment:
Don't yet have any other python apps, but once we did I may generalize some of this out more.